Sources/Files.swift

```diff
@@ -716,6 +725,15 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public func containsSubfolder(named folderName: String) -> Bool {
         return (try? subfolder(named: folderName)) != nil
     }
+
+    /**
+     *  Return whether this folder contains a given subfolder
+     *
+     *  - parameter subfolder: The folder to check for
+     */
+    public func contains(_ subfolder: Folder) -> Bool {
+        return subfolders.contains(subfolder)
```

Comment:
Same comment here as for when checking for a file.

Reply:
I changed the implementation to defer to `containsSubfolder(named:)`.